drivers/uktty/pl011/pl011.c

```diff
@@ -105,39 +103,63 @@ static void init_pl011(__u64 bas)
 
 	/* Just enable UART and data transmit/receive */
 	PL011_REG_WRITE(REG_UARTCR_OFFSET, CR_TXE | CR_UARTEN);
+
+	return 0;
 }
 
-void pl011_console_init(const void *dtb)
+int pl011_console_init(void *dtb)
```

Comment:
Why is the `const` removed?

Reply:
Honestly I don't remember, but the interface of this function is updated on a later commit anyway. If you think it's necessary I can revert this change though.

Reply:
Meh, if it's overwritten anyway I would say don't bother :shrimp: :fried_shrimp: :man_shrugging: